pkg/perf/cache.go

```diff
@@ -0,0 +1,49 @@
+package perf
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// TestResultData the result test schema
+type TestResultData struct {
```

Comment:
Now after checking the rest of the files, this should be TaskResult struct only task name and the result